clients/extension/scripts/popup.js

```diff
@@ -5,58 +5,10 @@
  */
 'use strict';
```

Comment:
FWIW we should probably rename this file to be `popup-entry.js` or whatever so it's the same clear signal that it's the root of a bundle like the others. I don't feel strongly about it today, though